docker/docker-compose.dev.yml

```diff
@@ -49,3 +50,12 @@ services:
       MB_IMPORT_DUMPS: "true"
     ports:
       - "5430:5432"
+
+  musicbrainz_test_db:
```

Comment:
Is it possible here to use https://hub.docker.com/r/metabrainz/musicbrainz-test-database/ ? (users already have it downloaded so that they can run the musicbrainz database)